### PR TITLE
Testing and docs for Action Blocks component

### PR DESCRIPTION
**Pull request by mikemai2awesome**
## Jira

http://vjira2:8080/browse/BDS-965

## Summary

Added testing for the Action Blocks component.

## Details

1. `action-blocks.js` in the tests folder now contains tests for all props.
2. `testing.md` now has detailed instructions on manual testing.
3. Folder structure is updated to follow conventions.
4. Certain demo pages were improved to have more clarity.

## How to test

Run the branch locally and read through the testing javascript and markdown.

### CODE REVIEW

File: packages/components/bolt-action-blocks/TESTING.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+2. Then, enable javascript and watch as the Action Blocks re-renders on the client-side.
+3. The layout should not shift, e.g. the spacing in between each action-blocks item should be identical before and after the web component re-renders.
+
+# Action Blocks component functional testing steps
+
+Functional testing should be performed manually by the QA team across the standard compliment of browsers. In each scenario, browser-type is specified when necessary. If browser type is not specified, the test applies to both "desktop" and "mobile" browsers.
+
+## Feature: action-blocks
+
+    In order to present items in a vertical or horizontal action-blocks
```
Comment (danielamorse) [inline packages/components/bolt-action-blocks/TESTING.md]: "items in ~~a~~ vertical or horizontal action-blocks"

File: packages/components/bolt-action-blocks/TESTING.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+1. Given I am viewing the [border variations page](https://boltdesignsystem.com/pattern-lab/patterns/02-components-action-blocks-15-action-blocks-border-variations/02-components-action-blocks-15-action-blocks-border-variations.html).
+2. If border is set to true, there should be a border in between items.
+3. If border is set to false, there should be no border in between items.
+
+## Scenario: theme variations
+
+1. Given I am viewing the [theme variations page](https://boltdesignsystem.com/pattern-lab/patterns/02-components-action-blocks-20-action-blocks-theme-variations/02-components-action-blocks-20-action-blocks-theme-variations.html).
+2. Inside an xlight theme, the Action Blocks should have xlight background.
+3. Inside an light theme, the Action Blocks should have light background.
```
Comment (danielamorse) [inline packages/components/bolt-action-blocks/TESTING.md]: "Inside a light"

File: packages/components/bolt-action-blocks/TESTING.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+4. Inside an dark theme, the Action Blocks should have dark background.
```
Comment (danielamorse) [inline packages/components/bolt-action-blocks/TESTING.md]: ditto

File: packages/components/bolt-action-blocks/TESTING.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+5. Inside an xdark theme, the Action Blocks should have xdark background.
```
Comment (danielamorse) [inline packages/components/bolt-action-blocks/TESTING.md]: ditto

File: packages/components/bolt-action-blocks/TESTING.md (hunk 1 above)
Comment (mikemai2awesome) [reply packages/components/bolt-action-blocks/TESTING.md]: Copy fixed.

File: packages/components/bolt-action-blocks/TESTING.md (hunk 2 above)
Comment (mikemai2awesome) [reply packages/components/bolt-action-blocks/TESTING.md]: Copy fixed.

File: packages/components/bolt-action-blocks/TESTING.md (hunk 3 above)
Comment (mikemai2awesome) [reply packages/components/bolt-action-blocks/TESTING.md]: Copy fixed.

**Comment by mikemai2awesome** [comment]
@danielamorse 

Changed the copy and pulled in master to resolve conflicts.

**Pull request merged**